src/main/scala/com/gu/fastly/Lambda.scala

```diff
@@ -30,16 +32,11 @@ class Lambda {
         case (ItemType.Content, EventType.Delete) =>
           sendFastlyPurgeRequestAndAmpPingRequest(event.payloadId, Hard, config.fastlyDotcomServiceId, makeDotcomSurrogateKey(event.payloadId), config.fastlyDotcomApiKey)
 
-        case (ItemType.Content, EventType.Update) =>
```

Comment:
Lets make it official that the 2 different representations of an Update should be treated the same.